products/compute/terraform.yaml

```diff
@@ -344,6 +349,9 @@ files: !ruby/object:Provider::Config::Files
     'google/transport_test.go': 'templates/terraform/transport_test.go'
     'google/import.go': 'templates/terraform/import.go'
     'google/import_test.go': 'templates/terraform/import_test.go'
+    # Handwritten acceptance tests for autogenerated resources.
+    # Adding them here allows updating the tests as part of a MM pull request.
+    'google/resource_compute_global_address_test.go': 'templates/terraform/tests/resource_compute_global_address_test.go'
```

Comment:
Try to keep the YAML files 80-chars. It works well if you break this line

Reply:
Done